docs/src/tutorial/example-data.rst

```diff
@@ -0,0 +1,89 @@
+Run the workflow using example data
+===================================
+
+In this tutorial, you will run the workflow using example reference data.
+
+.. contents:: Table of Contents
+   :local:
+
+Prerequisites
+-------------
+
+1. :doc:`setup`. These instructions will install all of the software you need to complete this tutorial and others.
+
+Setup
+-----
+
+1. Activate the ``nextstrain`` conda environment:
+
+   .. code:: text
+
+      conda activate nextstrain
+
+2. Change directory to the ``ncov`` directory:
+
+   .. code:: text
+
+      cd ncov
+
+3. Download the example tutorial repository into a new directory ``ncov-tutorial/``:
+
+   .. code:: text
+
+      $ git clone https://github.com/nextstrain/ncov-tutorial
+      Cloning into 'ncov-tutorial'...
+      [...more output...]
+
+Run the workflow
+----------------
+
+From within the ``ncov/`` directory, run the ``ncov`` workflow using a configuration file provided in the tutorial directory:
+
+.. code:: text
+
+   nextstrain build . --cores 4 --configfile ncov-tutorial/example-data.yaml
+
+Break down the command
+~~~~~~~~~~~~~~~~~~~~~~
+
+The workflow can take several minutes to run. While it is running, you can learn about the parts of this command:
+
+- ``nextstrain build .``
+   - This tells the :term:`docs.nextstrain.org:Nextstrain CLI` to :term:`build <docs.nextstrain.org:build (verb)>` the workflow from ``.``, the current directory. All subsequent command-line parameters are passed to the workflow manager, Snakemake.
+- ``--cores 4``
+   - This required Snakemake parameter specifies the number of CPU cores to use (`more info <https://snakemake.readthedocs.io/en/stable/executing/cli.html>`_).
+- ``--configfile ncov-tutorial/example-data.yaml``
+   - ``--configfile`` is another Snakemake parameter used to configure the ncov workflow.
+   - ``ncov-tutorial/example-data.yaml`` is a YAML file with contents (comments excluded):
+
+      .. code-block:: yaml
+
+         inputs:
+           - name: reference_data
+             metadata: https://data.nextstrain.org/files/ncov/open/reference/metadata.tsv.xz
+             sequences: https://data.nextstrain.org/files/ncov/open/reference/sequences.fasta.xz
+
+         refine:
+           root: "Wuhan-Hu-1/2019"
+
+      This provides the workflow with one input named ``reference_data``. The metadata and sequences files are downloaded directly from the associated URLs.
+
+      The ``refine`` entry specifies the root sequence for the example GenBank data.
+
+Visualize the results
+---------------------
+
+Run this command to start the :term:`docs.nextstrain.org:Auspice` server, providing ``auspice/`` as the directory containing output dataset files:
+
+.. code:: text
+
+   nextstrain view auspice/
+
+Navigate to ``http://127.0.0.1:4000/ncov/default-build``. The resulting :term:`docs.nextstrain.org:dataset` should show a phylogeny of ~200 sequences:
+
+.. figure:: ../images/dataset-example-data.png
```

Comment:
This screenshot shows the tree colored by clade, but the default color-by for this build is by country, so my tree from running the above commands looks like this instead:

<img width="986" alt="image" src="https://user-images.githubusercontent.com/85372/159589763-afbe9f77-fd39-4f6d-9c1c-4c47e8d078f4.png">

We should update the default Auspice config JSON to color by `clade_membership` by default, to make it match our own Nextstrain builds and this screenshot.

Reply:
993040395ade8796766e894813cb26f2329f01e1 should fix this.

Reply:
I think it makes sense for the default behavior for all users to match the defaults for our live SARS-CoV-2 builds, but I've just pinged Emma on Slack about this, since she originally changed from clade to country back in June 2020 (1c2dcdb20).

Reply:
Pulled this out into a separate PR which can get merged first: #897